pyproject.toml

```diff
@@ -46,7 +46,19 @@ documentation = "https://ietf-tools.github.io/xml2rfc/"
 issues = "https://github.com/ietf-tools/xml2rfc/issues"
 
 [project.optional-dependencies]
-pdf = ["weasyprint==65.0"]
+pdf = [
+    "dict2xml",
+    "weasyprint==65.0",
+]
+tests = [
+    "certifi",
+    "decorator",
+    "pyflakes",
+    "pypdf>=4.1.0",
+    "tox",
+    "tox-gh-actions",
+    "xml2rfc[pdf]",
```

Comment:
Not really sure this will archive what we want. This might end up install `xml2rfc` from PyPI. I think we should add `"weasyprint==65.0"` as a separate dependency to tests.

Reply:
aha. `pip` dependency resolution seems to do the right thing. If you are keen I'm okay with going ahead with `"xml2rfc[pdf]",`